std/algorithm/comparison.d

```diff
@@ -1567,3 +1567,156 @@ unittest
                 2, "two",
                 ));
 }
+
+/** Returns: First Argument (element of $(D a)) whose implicit conversion to
```

Comment:
Everything code must be enclosed in backquotes or $(D ...). That includes Argument, bool, true, etc.